centreon/www/front_src/src/components/FederatedComponents/index.tsx

```diff
@@ -46,7 +46,7 @@ const FederatedModules = ({
         )}
       </>
     ),
-    memoProps: [federatedModulesConfigurations]
+    memoProps: [federatedModulesConfigurations, rest]
```

Comment:
```suggestion
    memoProps: [federatedModulesConfigurations, ...(rest || undefined)]
```

Reply:
not worked for me i recieved error , and a blank page